src/components/reusuables/sections/CarouselSection.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import "react-responsive-carousel/lib/styles/carousel.min.css"
+import { Carousel } from "react-responsive-carousel"
+import FeatureWorkCard from "../cards/FeatureWorkCard"
+
+const CarouselSection = (props) => {
```

Comment:
Change props to the prop name. Example, (props)-> ({heading,subtitle})

Reply:
removed props